src/ducks/analysis/AnalysisTabs.jsx

```diff
@@ -26,7 +21,7 @@ const AnalysisTabs = () => {
             label={t(`Nav.${tabName}`)}
             key={tabName}
             name={tabName}
-            onClick={() => navigate(tabRoutes[tabName])}
+            onClick={() => navigate(`/analysis/${tabName}`)}
```

Comment:
y'a un `/` de trop avant `analysis` en rapport à ce qu'on avait avant. Je sais pas si c'est important ou non par contre... ? Je valide quand même pour te débloquer, je te laisse tester et faire un retour ici (et corriger si nécessaire) 👍

Reply:
Justement, c'est là qu'est le fix ! En v6, un chemin qui ne commence pas par un `/` est désormais systématiquement traité comme étant relatif (cf. https://reactrouter.com/en/main/upgrading/v5#note-on-link-to-values). Mais là, on veut un lien absolu, pas relatif, pour ne pas dépendre de la page sur laquelle sont affichés les onglets.